src/Swashbuckle.SwaggerGen/Annotations/SwaggerProducesResponseTypeAttribute.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Swashbuckle.SwaggerGen.Annotations
+{
+    /// <summary>
+    /// Response attribute that extends the standard <see cref="ProducesResponseTypeAttribute"/> allowing
+    /// additional Swagger annotation.
+    /// </summary>
+    /// <remarks>
+    /// <para>
+    /// This attribute derives from <see cref="ProducesResponseTypeAttribute"/> so you don't
+    /// have to put duplicate/redundant attributes on a controller or action to serve both Swagger
+    /// and standard ASP.NET API description purposes - just use the extended
+    /// Swagger response attribute and it works for both.
+    /// </para>
+    /// </remarks>
+    /// <seealso cref="Microsoft.AspNetCore.Mvc.ProducesResponseTypeAttribute" />
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = true, Inherited = true)]
+    public class SwaggerProducesResponseTypeAttribute : ProducesResponseTypeAttribute
```

Comment:
I like the idea of extending ProducesResponseType, but would prefer to still call the attribute _SwaggerResponseAttribute_ as it reads a little better and is more consistent with the existing _SwaggerOperationAttribute_